typescript/infra/scripts/funding/write-alert.ts

```diff
@@ -1,42 +1,46 @@
-import { checkbox } from '@inquirer/prompts';
-import yargs from 'yargs';
-
 import { ChainMap } from '@hyperlane-xyz/sdk';
 import { rootLogger } from '@hyperlane-xyz/utils';
 
-import { THRESHOLD_CONFIG_PATH } from '../../src/config/funding/balances.js';
+import {
+  BalanceThresholdType,
+  THRESHOLD_CONFIG_PATH,
+  ThresholdConfigs,
+  balanceThresholdConfigMapping,
+} from '../../src/config/funding/balances.js';
 import {
   AlertType,
   alertConfigMapping,
 } from '../../src/config/funding/grafanaAlerts.js';
+import { validateThresholds } from '../../src/funding/balances.js';
 import {
   fetchGrafanaAlert,
-  fetchServiceAccountToken,
+  fetchGrafanaServiceAccountToken,
   generateQuery,
   updateGrafanaAlert,
-} from '../../src/funding/grafana.js';
+} from '../../src/infrastructure/monitoring/grafana.js';
 import { readJSONAtPath } from '../../src/utils/utils.js';
-import { withAlertType, withConfirmAllChoices } from '../agent-utils.js';
 
 async function main() {
-  const { alertType, all } = await withConfirmAllChoices(
-    withAlertType(yargs(process.argv.slice(2))),
-  ).argv;
+  const saToken = await fetchGrafanaServiceAccountToken();
+
+  const balanceThresholdTypes = Object.values(BalanceThresholdType);
+  const balanceThresholdConfigs: ThresholdConfigs =
+    balanceThresholdTypes.reduce((acc, balanceThresholdType) => {
+      const thresholds = readJSONAtPath(
+        `${THRESHOLD_CONFIG_PATH}/${balanceThresholdConfigMapping[balanceThresholdType].configFileName}`,
+      ) as ChainMap<string>;
+
+      return {
+        ...acc,
+        [balanceThresholdType]: {
+          thresholds,
+        },
+      };
+    }, {} as ThresholdConfigs);
```

Comment:
since ThresholdConfigs's keys are just the BalanceThresholdType, maybe instead of the reduce logic I see in a few places, we can just do a map instead

easier to follow that we're just mapping from a threshold type to its corresponding values

```
const balanceThresholdConfigs: ThresholdConfigs = Object.fromEntries(
  balanceThresholdTypes.map((type) => [
    type,
    {
      thresholds: readJSONAtPath(
        `${THRESHOLD_CONFIG_PATH}/${balanceThresholdConfigMapping[type].configFileName}`,
      ) as ChainMap<string>,
    },
  ])
);
```